kotlin/src/main/kotlin/org/jetbrains/research/testspark/kotlin/KotlinPsiMethodWrapper.kt

```diff
@@ -68,6 +68,22 @@ class KotlinPsiMethodWrapper(val psiFunction: KtFunction) : PsiMethodWrapper {
         return lineNumber in startLine..endLine
     }
 
+    fun getInterestingPsiClassesWithQualifiedNames(): MutableSet<PsiClassWrapper> {
+        val interestingPsiClasses = mutableSetOf<PsiClassWrapper>()
+
+        psiFunction.valueParameters.forEach { parameter ->
+            val typeReference = parameter.typeReference
+            if (typeReference != null) {
+                val psiClass = PsiTreeUtil.getParentOfType(typeReference, KtClass::class.java)
+                if (psiClass != null && psiClass.fqName != null && !psiClass.fqName.toString().startsWith("kotlin.")) {
```

Comment:
A small comment on this if-statement also would be appreciated.

Reply:
actually no need for this `if`, because if null is there, then the psiClass will be null, as well and we will not add this parameter class to the result